analyses/mwi/dryspells_trigger/15_mwi_hdx_processing.md

```diff
@@ -0,0 +1,138 @@
+## MWI processing for HDX
+
+Code to process MWI ARC2 rainy season and dry spell data frames for upload to HDX.
+
+```python
+import pandas as pd
+import os
+import fuzzyjoin
+import numpy as np
+
+data_dir = os.environ["AA_DATA_DIR"]
+
+rainy_df_path = os.path.join(data_dir, 'public', 'exploration', 'mwi', 'arc2', 'mwi_arc2_rainy_seasons.csv') 
+rainy_df = pd.read_csv(rainy_df_path)
+
+dry_spell_path = os.path.join(data_dir, 'public', 'exploration', 'mwi', 'arc2', 'mwi_arc2_centroid_dry_spells.csv')
+dry_spell_df = pd.read_csv(dry_spell_path, index_col=0, parse_dates=['dry_spell_last_date', 'dry_spell_first_date', 'dry_spell_confirmation'])
```

Comment:
I understand that the rainy season onset/cessation dates and the dry spell first/last dates were computed elsewhere. Since you are computing the sum of rainfall during a dry spell on ln 104, am I correct in inferring that total_rainfall isn't a variable in `mwi_arc2_centroid_dry_spells.csv`?

Reply:
Yes, correct. Ideally would all be done in one script but hadn't realised we hadn't the total rainfall when originally aggregating so for ease of getting something out there put this together. Lots to refactor / improve in the next iteration!